### PR TITLE
JNG-5045 Support inline disposion

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5045" title="JNG-5045" target="_blank">StoryJNG-5045</a>  File preview in browser tab
  </td></table>
  <br />

### CODE REVIEW

**Review by noherczeg — APPROVED** [review]

File: filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java [hunk 1]
```diff
@@ -170,7 +171,11 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             String fileName = fileStoreService.getFileName(fileId);
             String contentType = fileStoreService.getMimeType(fileId);
             long size = fileStoreService.getSize(fileId);
-            final Optional<String> disposition = Optional.ofNullable(downloadToken).map(t -> (String) t.get(DownloadClaim.DISPOSITION));
+            Optional<String> disposition = Optional.ofNullable(inlineDispositionFromUrl);
```
Comment (bencegelei) [inline filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java:174]: 👀

**Review by bencegelei — APPROVED** [review]

File: filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java (hunk 1 above)
Comment (noherczeg) [reply filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java:174]: dfq!

Comment (bencegelei) [reply filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java:174]: https://discord.com/channels/918892501434241054/956479699406712863/1143966684617646160

Comment (bencegelei) [reply filestore-servlet/src/main/java/hu/blackbelt/osgi/filestore/servlet/DownloadServlet.java:174]: what-surprised

**Pull request merged**